### ISSUE TITLE
XHTML - add support for global references and identifiers

### ISSUE THREAD

**Issue by samnung** [epuber-io/epuber#44]
It would be nice, if the user don't have to specify path to file when creating link from one point to another in book. Makes some things easier.

There is a question, how it should work? Just when some referenced identifier can't be found in current file, will be searched in global scope? Or the identifier has to have prefix to make it working?

Also this requires different processing of all text files, since now it is processed only once in the same order as in spine, so how the user can link to file that is not yet processed?

**Comment by samnung** [comment]
This is already done in #57
